ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
+./ci/github/build_driver_container
+
+echo "Deploy KMMO..."
+make deploy KUSTOMIZE_CONFIG_DEFAULT=ci/install-ci
+
+
+echo "Wait until the KMMO Deployment is Available..."
+kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager --request-timeout=1m
+
+echo "Describe the Deployment and get its YAML if that failed..."
+kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+#        if: ${{ failure() }}
```

Comment:
I think we should move it to the infra part with all the other data gathered.

Also, IMO, we should call it only on failures.